cocos/ui/UIListView.cpp

```diff
@@ -93,7 +93,7 @@ void ListView::updateInnerContainerSize()
         case Direction::VERTICAL:
         {
             size_t length = _items.size();
-            float totalHeight = (length - 1) * _itemsMargin;
+            float totalHeight = ((float)length - 1) * _itemsMargin;
```

Comment:
_itemsMargin is float, so the (length-1) * _itemsMargin would also be float, right?
Why should we add a extra cast here?